### PR TITLE
[AudioLDM2] Doc fixes

### PR DESCRIPTION
**Pull request by sanchit-gandhi**
# What does this PR do?

Doc fixes:
1. Fix links to AudioLDM2 UNet / Projection model classes
2. Update checkpoint info with more details
3. Only use one end-to-end example (as recommended by @sayakpaul)

### CODE REVIEW

File: src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py [hunk 1]
```diff
@@ -51,19 +51,32 @@
...
+        >>> # set the seed
+        >>> generator = torch.Generator("cuda").manual_seed(0)
+
+        >>> # run the generation
+        >>> audio = pipe(
+        ...     prompt,
+        ...     negative_prompt=negative_prompt,
+        ...     num_inference_steps=200,
+        ...     audio_length_in_s=10.0,
+        ...     num_waveforms_per_prompt=3,
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py:75]: Let's maybe pass the generator if we define it
```suggestion
        ...     num_waveforms_per_prompt=3,
        ...     generator=generator,
```

File: src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py [hunk 2]
```diff
@@ -51,19 +51,32 @@
...
 
-        >>> prompt = "Techno music with a strong, upbeat tempo and high melodic riffs"
-        >>> audio = pipe(prompt, num_inference_steps=200, audio_length_in_s=10.0).audios[0]
+        >>> # define the prompts
+        >>> prompt = "The sound of a hammer hitting a wooden surface."
+        >>> negative_prompt = "Low quality."
 
-        >>> # save the audio sample as a .wav file
-        >>> scipy.io.wavfile.write("techno.wav", rate=16000, data=audio)
+        >>> # set the seed
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py:66]:
```suggestion
        >>> # set the seed for generator
```

**Review by patrickvonplaten — APPROVED** [review]
Small nit, apart from this looks good to me. Feel free to merge @sayakpaul

**Comment by sanchit-gandhi** [comment]
Good catch - thanks @patrickvonplaten 🙌

**Review by sayakpaul — APPROVED** [review]

**Pull request merged**